i18n/dictionaries/en-US.json

```diff
@@ -40,22 +40,22 @@
       }
     },
     "footer": {
-      "twitter": "Seguinos en Twitter",
-      "facebook": "Seguinos en Facebook",
-      "instagram": "Seguinos en Instagram",
-      "whatsapp": "Seguinos en Whatsapp",
-      "termsConditions": "Términos y condiciones",
-      "privacyPolicies": "Políticas de privacidad"
+      "twitter": "Follow us on Twitter",
+      "facebook": "Follow us on Facebook",
+      "instagram": "Follow us on Instagram",
+      "whatsapp": "Keep in touch by Whatsapp",
+      "termsConditions": "Terms and conditions",
+      "privacyPolicies": "Privacy Policies"
     },
     "header": {
-      "saveTimeMoney":"Ahorrá tiempo y dinero con tu tienda pency",
-      "publishCatalogOnline": "Publicá tu catálogo online y recibí por Whatsapp los pedidos de tus clientes ¡GRATIS!",
-      "createStore": "Crear tienda",
-      "seeTestShop": "Ver tienda de prueba"
+      "saveTimeMoney":"Save time and money with your Pency store",
+      "publishCatalogOnline": "Publish your catalog online and receive by WhatsApp your customers orders for FREE!",
```

Comment:
Creo que seria `your online catalog` y `your customers orders on WhatsApp for free`